tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/RcmBase.cs

```diff
@@ -26,6 +30,25 @@ public RcmBase(ITestOutputHelper outputHelper, string testName)
 
     protected string LogDirectory => Path.Combine(DatadogLogging.GetLogDirectory(), $"{GetTestName()}Logs");
 
+    internal static void CheckAckState(GetRcmRequest request, string product, uint expectedState, string expectedError, string message)
```

Comment:
Since you've added this, can you delete the `AspNetCore5AsmToggle.CheckAckState` method?

Reply:
Uhm, it may come from a rebase, but I'll look into it